mycli/sqlexecute.py

```diff
@@ -354,3 +358,40 @@ def reset_connection_id(self):
     def change_db(self, db):
         self.conn.select_db(db)
         self.dbname = db
+
+    def _create_ssl_ctx(self, sslp):
+        import ssl
+
+        ca = sslp.get("ca")
+        capath = sslp.get("capath")
+        hasnoca = ca is None and capath is None
```

Comment:
Should this use an `or` condition instead of `and` ?

Reply:
ca and capath don't need to be combined - if either of them is given, a custom certificate is being used. So only if neither are passed we use the system certificates (and don't verify).
As I said, the logic in this is taken from PyMySQL's [Connection._create_ssl_context](https://github.com/PyMySQL/PyMySQL/blob/dd47caae95011e79b9e2ee12549d23f05a7f839d/pymysql/connections.py#L361), so this is already how it worked before my changes.
I could try to contribute the changes inside `_create_ssl_context` to PyMySQL instead, but since it allows passing a SSLContext directly, this seemed easier.